examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+import os, time, h5py
```

Comment:
Warning: this data is bad due to either bad tuning or something. However, the matching of the time/power log with the enhancement and IR to get the averaged power works so this PR is primarily focused on that goal